### PR TITLE
Modify example app to work with FreeRTOS+IoT

### PR DESCRIPTION
**Pull request by TakayukiMatsuo**
# Description

Changed the sample application for Renesas board RSK+RX65N-2MB to run on FreeRTOS. Also provided instruction manuals(Engllish and Japanese).

Fixes zd#

# Testing

The sample application was run on the eva-board and tested for TLS communucation with example/server.

# Checklist

 - [ ] added tests
 - [ ] updated/added doxygen
 - [x ] updated appropriate READMEs
 - [x ] Updated manual and documentation

### CODE REVIEW

**Review by cconlon — CHANGES_REQUESTED** [review]

File: IDE/Renesas/e2studio/RX65N/RSK/README_EN.md [hunk 1]
```diff
@@ -3,241 +3,7 @@ wolfSSL sample application project for Renesas RSK+RX65N-2MB evaluation board
 
 <br>
 
-## 1. Overview
------
+A sample program for evaluating wolfSSL targeting the Renesas RX72N EnvisionKit evaluation board is provided. For details on the program, refer to the following documents included in the package.
```
[the remark this answers is not in the document]
Comment (TakayukiMatsuo) [reply IDE/Renesas/e2studio/RX65N/RSK/README_EN.md]: Fixed it. Unfortunately, Renesas only provides the page for RX72N, so I use the reference there as it is.

**Comment by TakayukiMatsuo** [comment]
Renesas will add the board definition file to be provided. The name is still tentative. I will wait for them to release the file after February 28th and reflect it in the instruction manual. Please keep this PR open until I'm done.

**Comment by TakayukiMatsuo** [comment]
Since Renesas released a new board definition file on the web, instruction manuals were updated to renew some snap shots.
Please resume the review.

**Review by cconlon — APPROVED** [review]

**Comment by cconlon** [comment]
@TakayukiMatsuo Does your last comment mean that we can go ahead and merge this PR now (upstream files are in place and ready)?

**Comment by TakayukiMatsuo** [comment]
Yes, it is. Since Renesas released a new board config file on the web earlier than planned and I have reflected that filename and snap shots in the manuals, we need not to wait the due date.

**Pull request merged**